Dockerfile

```diff
@@ -29,13 +29,20 @@ WORKDIR /aleph
 ENV PYTHONPATH /aleph
 RUN pip install -q -e /aleph
 
+RUN mkdir -p /opt/ftm-compare/word-frequencies/ && \
+    curl -L -o "/opt/ftm-compare/word-frequencies/word-frequencies.zip" "https://public.data.occrp.org/develop/models/word-frequencies/word_frequencies.zip" && \
+    python3 -m zipfile --extract /opt/ftm-compare/word-frequencies/word-frequencies.zip /opt/ftm-compare/word-frequencies/ && \
+    curl -L -o "/opt/ftm-compare/model.pkl" "https://public.data.occrp.org/develop/models/xref/glm_bernoulli_2e_wf-v0.4.1.pkl"
```

Comment:
I'm assuming the model is versioned to match FTM-compare? Are we going to generate a new model for each new version? Should we expect any weird behaviour if the library version doesn't match the model version?

How often are we going to generate the word frequencies? Should we version those as well, may be with a timestamp?

Reply:
Good questions. New models don't _need_ to be generated for each ftm-c version, but I expect many of the ftm-c commits being related to model updates which would require a model rebuild.

The way I was thinking about model version matching is:
- Major version must match
- Minor version is optional to match, but you may be missing out on a feature
- Patch versions are irrelevant

Word frequencies should probably also be labeled with the ftm-c version they were built with for the same reason.

For all of these static files, I imagine they only need to be rebuilt once per quarter at most. The word frequency file only needs to be updated if the entities in aleph significantly change and the model file only needs to be updated if the xref decisions on aleph significantly change.